client/my-sites/plan-features-2023-grid/plan-comparison-grid.tsx

```diff
@@ -764,20 +762,11 @@ export const PlanComparisonGrid: React.FC< PlanComparisonGridProps > = ( {
 			<PlanComparisonHeader className="wp-brand-font">
 				{ translate( 'Compare our plans and find yours' ) }
 			</PlanComparisonHeader>
-			<PlanTypeSelector
+			<TermExperimentPlanTypeSelector
+				isEligible={ true }
 				kind="interval"
 				plans={ displayedPlansProperties.map( ( { planName } ) => planName ) }
-				isInSignup={ planTypeSelectorProps.isInSignup }
-				isStepperUpgradeFlow={ planTypeSelectorProps.isStepperUpgradeFlow }
-				eligibleForWpcomMonthlyPlans={ planTypeSelectorProps.eligibleForWpcomMonthlyPlans }
-				isPlansInsideStepper={ planTypeSelectorProps.isPlansInsideStepper }
-				intervalType={ planTypeSelectorProps.intervalType }
-				customerType={ planTypeSelectorProps.customerType }
-				hidePersonalPlan={ planTypeSelectorProps.hidePersonalPlan }
-				basePlansPath={ planTypeSelectorProps.basePlansPath }
-				siteSlug={ planTypeSelectorProps.siteSlug }
-				hideDiscountLabel={ false }
```

Comment:
The `hideDiscountLabel` prop is not passed in the refactored `TermExperimentPlanTypeSelector`, but this should be okay since we don't explicitly check for the `false` value. No action is required here, just adding this for the benefit of other reviewers. 🙂 
https://github.com/Automattic/wp-calypso/blob/abaa8eee9298e2c39623bb0e50c5cee4e306f212/client/my-sites/plans-features-main/plan-type-selector.tsx#L184

Reply:
Thanks for noting this, @aneeshd16 . I should have written an inline comment so there was no confusion :)